XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,573 @@
+
+*Note: These definitions do not directly match the definitions listed in the [W3C Verifiable Credentials spec](https://www.w3.org/TR/vc-data-model-2.0/#terminology). These terms are used in a slightly different way in the context of this spec.* 
+
+### 1.3. Basic Flow
+
+In this scenario, an authorizer/verifier account, Verity, is a regulated entity wants to ensure that she is only interacting with properly KYC'd accounts.
+
+* Verity sets up her account such that she only wants to interact with properly KYC'd accounts. He trusts Isabel, a credential issuer, to properly vet accounts and issue credentials attesting to that.
+* Alice, who wants to interact with Verity's account, submits her KYC documents to Isabel off-chain. Isabel creates a credential attesting to her trustworthiness on-chain.
+* Alice accepts the credential, making it a valid credential.
+* Alice can now interact with/send funds to Verity.
+
+## 2. On-Ledger Object: `Credential`
+
+This object will be an on-chain representation of a credential, while ensuring that any sensitive details are kept off-chain, to protect the subject's privacy.
+
+### 2.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`LedgerEntryType`| ✔️|`string`|`UInt16`|The ledger object's type (`Credential`).|
+|`Flags`| ✔️|`number`|`UInt32`|Flag values associated with this object.|
+|`Subject`| ✔️|`string`|`AccountID`|The account that the credential is for.|
+|`Issuer`| ✔️|`string`|`AccountID`|The issuer of the credential.|
+|`CredentialType`| ✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
+|`Expiration`| |`number`|`UInt32`|Optional credential expiration.|
+|`URI`| |`string`|`Blob`|Optional additional data about the credential (such as a link to the VC document).|
+|`OwnerNode`|✔️|`string`|`UInt64`|A hint indicating which page of the sender's owner directory links to this object, in case the directory consists of multiple pages. Note: The object does not contain a direct link to the owner directory containing it, since that value can be derived from the `Account.PreviousTxnID`.|
+|`PreviousTxnID`|✔️|`string`|`Hash256`|The identifying hash of the transaction that most recently modified this object.|
+|`PreviousTxnLgrSeqNumber`|✔️|`number`|`UInt32`|The index of the ledger that contains the transaction that most recently modified this object.|
+
+#### 2.1.1. Object ID
+
+The ID of this object will be a hash that incorporates the `Subject`, `Issuer`, and `CredentialType` fields, combined with a unique space key for `Credential` objects, which will be defined during implementation.
+
+#### 2.1.2. `Flags`
+| Flag Name | Flag Value |
+|-----------|------------|
+|`lsfAccepted`|`0x00010000`|
+
+`lsfAccepted` is on if the subject of the credential has accepted the credential. If it is off, the issuer has the reserve burden; if it is on, the burden moves to the subject.
```

Comment:
Typically in the docs we use the words "enabled" and "disabled" for flags instead of "on" and "off". Not a big deal either way though.

Instead of saying "burden", I would say "responsible" for the reserve here:

```suggestion
The `lsfAccepted` flag represents whether the subject of the credential has accepted the credential. If this flag is disabled, the issuer is responsible for this ledger entry's reserve; if the flag is enabled, the subject of the credential is responsible for the reserve instead. This flag is disabled by default, but is enabled as the result of a successful CredentialAccept transaction.
```